source/plugins/ruby/KubernetesApiClient.rb

```diff
@@ -751,8 +751,13 @@ def getMetricNumericValue(metricName, metricVal)
             metricValue.chomp!("k")
             metricValue = Float(metricValue) * 1000.0 ** 1
           elsif (metricValue.end_with?("m"))
+            #original value before downcase ending with M is megabyte and value ending with m is milli-byte
             metricValue.chomp!("m")
-            metricValue = Float(metricValue) * 1000.0 ** 2
```

Comment:
also, can you check if Ux (live) supportes this unit ? if not we need to fix that as well.

Reply:
sure